test/src/tests-kit-blocking.ts

```diff
@@ -1427,21 +1427,17 @@ describe('kit blocking', () => {
                 window.mParticle.config.kitConfigs.push(forwarderDefaultConfiguration('MockForwarder'));
                 window.mParticle.init(apiKey, window.mParticle.config);
 
-                window.mParticle.logEvent('Blocked event');
+                await waitForCondition(hasIdentifyReturned);
```

Comment:
Should these have changed positions?

Reply:
technically it doesn't matter. i swapped the two just to show the tests still pass, but the reason it doesn't matter is the setTimeout previously just ensured the logged event wasn't in the queue and actually went to the forwarders.  so whether it is prior to the event being logged (so that it doesn't reach the queue), or after the event is logged (in which case the event goes into the queue, and the await flushes it out), it doesn't matter

Reply:
Works for me.